### PR TITLE
Remove duplicate entry from .gitignore.

### PR DESCRIPTION
**Pull request by jbampton**

### CODE REVIEW

File: docs/guide-forms.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ First we will start by adding the form to our template tag:
   </p>
   <div class="sender-info">
     <div>
-      <label for="name" class="label" >Your name</label>
+      <label for="name" class="label">Your name</label>
```
Comment (Mudassar045) [inline docs/guide-forms.md]: Changes should conform the PR title. Single space it's not a big issue. I would suggest `eslint` should be the part of `gridsome`

File: .gitignore [hunk 2]
```diff
@@ -4,7 +4,6 @@
 src/.temp
 node_modules
 dist
-.cache
```
[the remark this answers is not in the document]
Comment (mtlynch) [reply .gitignore:7]: It's a duplicate of line 3

**Comment by mtlynch** [comment]
@jbampton - Is this PR still live? It includes a stray whitespace change in `docs/guide-forms.md` that's blocking the merge.

**Comment by jbampton** [comment]
Hey @mtlynch yes this is still live.  And I removed the whitespace too 👍

**Comment by mtlynch** [comment]
Thanks, @jbampton!

**Pull request merged**